### PR TITLE
[DOCFIX] Improve clarity and organization of Docker docs

### PR DESCRIPTION
**Pull request by aaudiber**
Improved the separation between the tutorial and non-tutorial topics

### CODE REVIEW

File: docs/en/Running-Alluxio-On-Docker.md [hunk 1]
```diff
@@ -209,5 +161,45 @@ $ docker run -d --net=host --shm-size=1G \
...
+of having to go through the worker process. A limitation of this approach is that it doesn't play
+well with setting worker container memory limits. Even though the ramdisk is mounted on the host,
+writes to it from the worker container will count towards the container's memory limit. To work
+around this limitation, you can share a domain socket instead of the ramdisk itself. The downside
+of using domain sockets is that they are computationally more expensive and suffer from lower
+write throughput.
+
+## Domain Socket
+
+From the host machine
```
Comment (gpang) [inline docs/en/Running-Alluxio-On-Docker.md]: Maybe "From the host machine, create a directory for the shared domain socket."?

File: docs/en/Running-Alluxio-On-Docker.md [hunk 2]
```diff
@@ -104,29 +78,31 @@ $ docker run -d --net=host \
              alluxio master
 ```
 
-### Run the Alluxio worker
+- `-v $PWD/underStorage:/underStorage`: Share the underStorage folder with the Docker container.
```
[the remark this answers is not in the document]
Comment (aaudiber) [reply docs/en/Running-Alluxio-On-Docker.md]: done

File: docs/en/Running-Alluxio-On-Docker.md [hunk 3]
```diff
@@ -104,29 +78,31 @@ $ docker run -d --net=host \
...
-`/mnt/ramdisk` path on the host machine to the `/opt/ramdisk` path in the worker container.
-This way, the data written by the Alluxio worker can be directly accessed from outside the
-container.
+### Run the Alluxio worker
 
 ```bash
 $ # This gets the public ip of the current EC2 instance
 $ export INSTANCE_PUBLIC_IP=$(curl http://169.254.169.254/latest/meta-data/public-ipv4)
-$ docker run -d --net=host \
+$ ALLUXIO_WORKER_CONTAINER_ID=$(docker run -d --net=host \
```
[the remark this answers is not in the document]
Comment (aaudiber) [reply docs/en/Running-Alluxio-On-Docker.md]: we use it later in the doc, I've added a comment right before the command

File: docs/en/Running-Alluxio-On-Docker.md [hunk 4]
```diff
@@ -104,29 +78,31 @@ $ docker run -d --net=host \
...
              -v $PWD/underStorage:/underStorage \
              -e ALLUXIO_MASTER_HOSTNAME=${INSTANCE_PUBLIC_IP} \
              -e ALLUXIO_RAM_FOLDER=/opt/ramdisk \
              -e ALLUXIO_WORKER_MEMORY_SIZE=1GB \
              -e ALLUXIO_UNDERFS_ADDRESS=/underStorage \
-             alluxio worker
+             alluxio worker)
 ```
 
+- `-v /mnt/ramdisk:/opt/ramdisk`: Share the host machine's ramdisk with the Docker container.
```
[the remark this answers is not in the document]
Comment (aaudiber) [reply docs/en/Running-Alluxio-On-Docker.md]: done

File: docs/en/Running-Alluxio-On-Docker.md (hunk 1 above)
Comment (aaudiber) [reply docs/en/Running-Alluxio-On-Docker.md]: done

**Comment by aaudiber** [comment]
@gpang Thanks for the review! I've addressed the comments, PTAL

**Pull request merged**